cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -256,7 +264,7 @@
   if ( document.body.parentElement.className.indexOf( 'no-js' ) === -1 ) {
     !function(){
     {# Include site-wide JavaScript. #}
-    var a = [ '{{ static('js/routes/common.js') }}' ];
+    var s = [ '{{ static('js/routes/common.js') }}' ];
```

Comment:
Minor: should be indented one more level.

Reply:
Fixed